### DISCUSSION TITLE
2024-08-02 금요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#175] [기록보드]
2024-08-02 금요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 기술면접 준비 | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 16:30 ~ 18:00 | 프로젝트 집단 면접 | |
| 18:40 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 마이페이지 최종 완성
  - 퍼블리싱 작업
  - 내 프로필 / 타인 프로필 구분 조건부 렌더링

### 트러블 슈팅

- 시간 부족으로 코드 품질 높이지 못하고 컴포넌트 분리 못하고 있음

### 내일 계획

- 버디 프로필 수정 기능 구현

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 전체적인 스타일링 및 점검

### 트러블 슈팅

- 시간이 부족하당 ㅠㅠ

### 내일 계획

- 전수점검 내용 반영 작업

**Comment by slsl2** [comment]
## 개인 기록보드 서식 (김민지)

### 오늘 한 일

- 채팅 페이지 데이터 연결 중
-  home 페이지 배너 작업

### 트러블 슈팅

- 진전이 많이 없었어서 주말에 많이 해야 할 것 같습니다

### 주말 계획

- MVP까지 필요한 부분 완성
